src/test/scala/uk/gov/hmrc/test/ui/pages/CommonElements.scala

```diff
@@ -75,6 +75,13 @@ trait CommonElements extends BasePage with Matchers{
     "Year"                            -> By.id("amount-year"),
     "Pay"                             -> By.id("amount"),
     "Tax"                             -> By.id("amount"),
+    "1st change"                      -> By.cssSelector("#main-content > div > div > div:nth-child(3) > ul > li:nth-child(1) > span.hmrc-add-to-a-list__change > a"),
```

Comment:
could this be more detailed? e.g. "first change employment link" and "third remove employment link"